doc/release/README.compiling

```diff
@@ -37,8 +37,7 @@ We note some of the most useful flags here:
 
      -O : turns on optimization of the generated C code
 
-     --fast : turns on --no-checks, -O, and relaxes IEEE floating
-              point strictness
+     --fast : turns on --no-checks, -O, and enables many other optimizations
```

Comment:
What "many other optimizations" are being referred to here?  I can't think of any at all (and, generally, don't like vague characterizations like this).